Fix the bitcoinaverage.com API & etherchain.org API

ajslater opened the pull request:
The old bitcoinaverage.com API returns 404.

API v2 docs are here: https://apiv2.bitcoinaverage.com/#historical-data

ajslater committed:
fix the bitcoinaverage api

amacneil commented:
Thanks! Did you test? It looks like that endpoint returns json, not csv, so some other code might need updating.

ajslater committed:
bad extra character

ajslater commented:
My testing seemed to work, but now i suspect only because the url didn't 404

There was a 1 character error I've corrected, it now returns csv.


On Wed, Dec 20, 2017 at 12:00 PM, Adrian Macneil <notifications@github.com>
wrote:

> Thanks! Did you test? It looks like that endpoint returns json, not csv,
> so some other code might need updating.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/amacneil/cryptogains/pull/1#issuecomment-353167439>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAQFRjkxu1pnr8TH8t85vKbmaVgHPuohks5tCWdLgaJpZM4Qz5r6>
> .
>

ajslater committed:
etherchan changed API as well.

ajslater commented:
etherchain.org also changed their API
That change has been pushed.

It now runs well again


On Wed, Dec 20, 2017 at 2:38 PM, AJ <aj@slater.net> wrote:

> My testing seemed to work, but now i suspect only because the url didn't
> 404
>
> There was a 1 character error I've corrected it now returns csv.
>
>
> On Wed, Dec 20, 2017 at 12:00 PM, Adrian Macneil <notifications@github.com
> > wrote:
>
>> Thanks! Did you test? It looks like that endpoint returns json, not csv,
>> so some other code might need updating.
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/amacneil/cryptogains/pull/1#issuecomment-353167439>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/AAQFRjkxu1pnr8TH8t85vKbmaVgHPuohks5tCWdLgaJpZM4Qz5r6>
>> .
>>
>
>
